sky/backends/cloud_vm_ray_backend.py

```diff
@@ -4481,6 +4484,19 @@ def set_autostop(self,
                     # For SkyServe controllers on Kubernetes: override autostop
                     # behavior to force autodown (instead of no-op)
                     # to avoid dangling controllers.
+
+                    # down = False is the default, but warn the user in case
+                    # they have explicitly specified it.
+                    config_override_down = skypilot_config.get_nested(
+                        (controller.value.controller_type, 'controller',
+                         'autostop', 'down_when_idle'), None)
+                    if config_override_down is False:  # will not match None
+                        logger.warning(
+                            'SkyServe controller autodown is disabled in the '
+                            '~/.sky/config.yaml configuration file '
```

Comment:
```suggestion
                            '~/.sky/skyconfig.yaml configuration file '
```
due to a change from recently merged PR

Reply:
ok I lied we need to change this back to ~/.sky/config.yaml`